account_fiscal_position_rule/views/account_fiscal_position_rule_view.xml

```diff
@@ -1,220 +1,231 @@
-            <separator orientation="vertical"/>
-            <field name="name"/>
-            <field name="from_country"/>
-            <field name="from_state"/>
-            <field name="to_invoice_country"/>
-            <field name="to_invoice_state"/>
-          </group>
-          <newline/>
-        </search>
-      </field>
-    </record>
-
-    <!--  Wizard position fiscal rule -->
-    <record id="view_wizard_account_fiscal_position_rule" model="ir.ui.view">
-      <field name="name">Generate Fiscal Positions Rules from Templates</field>
-      <field name="model">wizard.account.fiscal.position.rule</field>
-      <field name="arch" type="xml">
-        <form string="Generate Fiscal Positions Rules from Templates" version="7.0">
-          <group string="This will automatically configure all fiscal position rules">
-            <field name="company_id"/>
-          </group>
-          <footer>
-            <button name="action_create" string="Create" type="object" class="oe_highlight"/> or
-	    <button special="cancel" string="Cancel" class="oe_link"/>
-          </footer>
-        </form>
-      </field>
-    </record>
-
-    <record id="action_wizard_account_fiscal_position_rule" model="ir.actions.act_window">
-      <field name="name">Generate Fiscal Positions Rules from Templates</field>
-      <field name="type">ir.actions.act_window</field>
-      <field name="res_model">wizard.account.fiscal.position.rule</field>
-      <field name="view_type">form</field>
-      <field name="view_mode">form</field>
-      <field name="target">new</field>
-    </record>
-
-    <menuitem parent="account.account_template_folder" action="action_wizard_account_fiscal_position_rule" id="menu_wizard_fiscal_position_rule"/>
-
-    <record id="action_account_fiscal_position_rule_form" model="ir.actions.act_window">
-      <field name="name">Fiscal Position Rule</field>
-      <field name="res_model">account.fiscal.position.rule</field>
-      <field name="view_type">form</field>
-      <field name="view_mode">tree,form</field>
-    </record>
-
-    <menuitem id="menu_action_account_fiscal_position_rule_form" action="action_account_fiscal_position_rule_form" parent="account.next_id_27" sequence="21"/>
+	<data>
+
+		<!-- Create Menu for Configuration and Generation Templates -->
+		<menuitem id="menu_account_fiscal_position_rule_template"
+				  parent="account.menu_finance_configuration" name="Fiscal Position Rule" sequence="21"/>
+
+		<!-- Fiscal Position Rule Template -->
+		<record id="view_account_fiscal_position_rule_template_form" model="ir.ui.view">
+			<field name="name">account.fiscal.position.rule.template.form</field>
+			<field name="model">account.fiscal.position.rule.template</field>
+			<field name="arch" type="xml">
+				<form string="Fiscal Position Rule Template">
+					<group>
+						<group string="General" colspan="4">
```

Comment:
I suggest put the name attribute in the groups, in version 9 we can't use the string attribute to inherit views anymore.